a.core_concepts.md

```diff
@@ -196,25 +196,22 @@ kubectl get po nginx -o jsonpath='{.spec.containers[].image}{"\n"}'
 </p>
 </details>
 
-### Get nginx pod's ip created in previous step, use a temp busybox image to wget its '/'
+### Get nginx pod's IP created in previous step, use a temporary BusyBox image to retrieve the welcome page
```

Comment:
Might give less away?

Reply:
Hm, this would require the reader to know that welcome page is '/'. Which, of course, it's the obvious, still I'm not 100% sure.